README.md

```diff
@@ -156,7 +156,10 @@ Before you begin using this Terraform module, ensure you meet the following prer
 | lb\_access\_logs\_lifecycle\_rule\_status | Define status of the S3 lifecycle rule. Possible options are enabled or disabled. | `string` | `"Disabled"` | no |
 | lb\_enable\_access\_logs | Enable or disable access logs for the NLB | `bool` | `false` | no |
 | lb\_access\_logs\_expiration\_days | Define the days after which the LB access logs should be deleted. | `number` | `14` | no |
-| bucket\_replication\_destination\_region | Define in which Region should the bucket be replicated | `string` | n/a | yes |
+| bucket\_replication\_destination\_region | Define in which Region should the bucket be replicated | `string` | `null` | no |
+| asg\_enable\_instance\_refresh | Enables instance refresh for the GraphDB Auto scaling group. A refresh is started when any of the following Auto Scaling Group properties change: launch\_configuration, launch\_template, mixed\_instances\_policy | `bool` | `true` | no |
+| asg\_instance\_refresh\_checkpoint\_delay | Number of seconds to wait after a checkpoint. | `number` | `3600` | no |
```

Comment:
Maybe it will be good to rename it to graphdb_asg_enable_instance_refresh? Like in the other module for example variables which are related to the LB are named lb_enable_access_logs and etc.

Reply:
Well the module is graphdb so it's pretty obvious it's graphdb's asg.